File: MORM.Java/src/MORM/Java/Models/Venda/Municipio.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "MUNICIPIO")
public class Municipio extends CollectionItem
{
    @CampoAttribute(Campo = "ID_MUNICIPIO", Tipo = CampoTipo.Key)
    public Integer Id_Municipio;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "CD_MUNICIPIO", Tipo = CampoTipo.Req)
    public Integer Cd_Municipio;
    @CampoAttribute(Campo = "DS_MUNICIPIO", Tipo = CampoTipo.Req)
    public String Ds_Municipio;
    @CampoAttribute(Campo = "DS_SIGLA", Tipo = CampoTipo.Req)
    public String Ds_Sigla;
    @CampoAttribute(Campo = "ID_ESTADO", Tipo = CampoTipo.Req)
    public Integer Id_Estado;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Municipio.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("MUNICIPIO")]
    public class Municipio : CollectionItem
    {
        [Campo("ID_MUNICIPIO", CampoTipo.Key)]
        public int Id_Municipio { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_MUNICIPIO", CampoTipo.Req)]
        public int Cd_Municipio { get; set; }
        [Campo("DS_MUNICIPIO", CampoTipo.Req)]
        public string Ds_Municipio { get; set; }
        [Campo("DS_SIGLA", CampoTipo.Req)]
        public string Ds_Sigla { get; set; }
        [Campo("ID_ESTADO", CampoTipo.Req)]
        public int Id_Estado { get; set; }
    }
}